### PR TITLE
Documentation for MongoDB

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: distribution/index.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ The easiest and most powerful way to get started is to download the API Platform
 It contains:
 
 * an API skeleton, including with [the server-side component](../core/index.md), [the Symfony 4
-  microframework](https://symfony.com/doc/current/setup/flex.html) and [the Doctrine ORM](http://docs.doctrine-project.org/projects/doctrine-orm/)
+  microframework](https://symfony.com/doc/current/setup/flex.html), [the Doctrine ORM](http://docs.doctrine-project.org/projects/doctrine-orm/)
```
Comment (dunglas) [inline distribution/index.md]: We'll not "bloat" the distribution with MongoDB support by default, can you revert this one?

**Comment by alanpoulain** [comment]
@vincentchalamon could you review this one please?

File: core/mongodb.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+If the mongodb PHP extension is not installed yet, [install it beforehand](https://secure.php.net/manual/en/mongodb.installation.pecl.php).
+
+If you are using the [API Platform Distribution](../distribution/index.md), modify the Dockerfile to add the extension:
+
+```diff
+ // api/Dockerfile
+
+ ...
+ 	pecl install \
```
Comment (dunglas) [inline core/mongodb.md]:
```suggestion
 	docker-php-ext-install \
```

Right?

Comment (alanpoulain) [reply core/mongodb.md]: Doesn't seem to be possible:
```shell
docker-php-ext-install -j4 intl mongodb pdo_pgsql zip
error: /usr/src/php/ext/mongodb does not exist
```
It's only available with PECL.

**Pull request merged**